### PR TITLE
Update travis Node.js version & remove io.js

### PR DESCRIPTION
**Pull request by demohi**

### CODE REVIEW

**Comment by dead-horse** [comment]
i think we'd better keep iojs in travis

**Comment by demohi** [comment]
@dead-horse  I added it back

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ node_js:
   - "0.12"
   - "iojs-v1"
   - "iojs-v2"
-  - "iojs-v3"
+  - "iojs-v3"
```
Comment (dougwilson) [inline .travis.yml]: Looks like there is some accidental trailing whitespace added to the v3 line.

Comment (demohi) [inline .travis.yml]: @dougwilson  yes..   thanks

**Comment by dead-horse** [comment]
can you squash(`git rebase master`) these commits?

**Comment by targos** [comment]
Wouldn't it be better to put `4` so that it automatically installs the latest 4.x?

**Comment by fengmk2** [comment]
landed 520163fe57e8ea6888e6116f2ea841213608eff3

**Pull request closed without merging**